molgenis-dataexplorer/src/main/resources/js/dataexplorer.js

```diff
@@ -541,6 +541,13 @@ $.when($,
                 $(document).trigger('changeQuery', createEntityQuery());
             });
 
+            $('#observationset-search').keypress(function (event) {
+                if (event.which == 13) {
+                    $(document).trigger('changeQuery', createEntityQuery());
+                    $("#observationset-search").change();
+                }
+            });
+
```

Comment:
Won't this fire two concurrent queries in browsers that do trigger when the enter button is pressed?